giraffe/webpack.config.js

```diff
@@ -70,4 +80,9 @@ module.exports = {
       },
     ],
   },
+  plugins: [
+    new webpack.optimize.LimitChunkCountPlugin({
+      maxChunks: 1,
+    }),
```

Comment:
what's up with the chunking optimization? does leaflet do this by default?

Reply:
This is a workaround for a webpack problem - once you start using dynamic imports it forces chunking of its output. There might be other conditions when this gets triggered, so I think it is correct to declare that giraffe is just one piece now.